ui_framework/src/components/header/_header.scss

```diff
@@ -0,0 +1,158 @@
+// Header. Includes breadcrumbs of nav buttons.
+
+$kuiHeaderChildSize: $kuiSizeXXL + $kuiSizeL;
+
+@include component('kuiHeader') {
+  background: $kuiHeaderBackgroundColor;
+  border-bottom: $kuiBorderThin;
+
+  @include child('logo') {
```

Comment:
Let's talk through this one and figure something out. I dislike breaking up components (on the sass) if they're not used outside of the component. I don't think this will get used anywhere else other than the header? Same with the breadcrumbs. They are couple pretty directly with the rest of the markup there (share same sizing vars..etc).

From the react side it makes sense though.